_includes/tutorials/multi-joins/ksql/code/src/statements.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE customers (customerid STRING PRIMARY KEY, customername STRING)
+    WITH (KAFKA_TOPIC='customers',
+          VALUE_FORMAT='json',
+          PARTITIONS=1);
+
+CREATE TABLE items (itemid STRING PRIMARY KEY, itemname STRING)
+    WITH (KAFKA_TOPIC='items',
+          VALUE_FORMAT='json',
+          PARTITIONS=1);
+
+CREATE STREAM orders (orderid STRING KEY, customerid STRING, itemid STRING, purchasedate STRING)
+    WITH (KAFKA_TOPIC='orders',
+          VALUE_FORMAT='json',
+          PARTITIONS=1);
+
+CREATE STREAM orders_enriched AS 
+  SELECT customers.customerid AS customerid, customers.customername AS customername, 
+         orders.orderid, orders.purchasedate,
+         items.itemid, items.itemname
+  FROM orders 
+  LEFT JOIN customers on orders.customerid = customers.customerid 
```

Comment:
Is it helpful to explain what `LEFT JOIN` is?  Or link out to something else that does explain it?

Reply:
I included a link in the tutorial to the ksqldb docs on joins: "See the ksqlDB documentation for the full details on joins".

IMO getting into the anatomy of joins in the text of the tutorial seems out of scope for this 'lesson', but just an opinion.   Does that link help?